sdk/kusto/test-resources.bicep

```diff
@@ -0,0 +1,85 @@
+param id string
```

Comment:
As we have deprecated track 1 SDK, I'd like to know why we still need all these changes in `Microsoft.Azure.Management.Kusto` directory.

Reply:
I'm not accentually sure what these are for, but I don't think it's in the Microsoft.Azure.Management.Kusto directory.

Reply:
Oh, sorry, I accidentally misread it

Let me clarify this, for this PR, I think the changes that we need should only happen in the directory `sdk/kusto/Azure.ResourceManager.Kusto`, could we revert other changes?

Reply:
As your test framework mentions https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/TestResources/New-TestResources.ps1.md, it is possible to setup an environment for the tests by using the New-TestResources.ps1 command feeding it a bicep file. The location of the bicep file is expected to be under sdk/libraryName directory. I'm not sure why this is problematic.

Reply:
This file is for our personal use. It is run once before recording. The point of running this file is to create a setup script that creates all the resources we need to create a test environment without having to write a bunch of code to create it.

Reply:
Ok, that makes sense to me, thanks for you explanation.